lib/index.js

```diff
@@ -326,6 +325,49 @@
     // Changed to stroke to match other stroke functions
     this.setStrokeDash = this.setLineDash;
 
+    function lastArgIs (fn, args) {
```

Comment:
You want to say `lastArgMatches(` to indicate that the expected first arg here is a **predicate** not a **value**.